src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -4,6 +4,10 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define AGE_REQ_ADULT 0
+#define AGE_REQ_CHILD 1
```

Comment:
since those are tested for equality against `gSaveContext.linkAge` it would be good to keep in mind using the same define for these and wherever `gSaveContext.linkAge` is used
eg with your other PR #775

https://github.com/zeldaret/oot/blob/a2bb615b507da53772ad040ed7207a1b4efac341/include/macros.h#L29-L33

->

```c
#define AGE_FLAG_ADULT 0
#define AGE_FLAG_CHILD 1
```

```c
 #define LINK_IS_CHILD (gSaveContext.linkAge != AGE_FLAG_ADULT) 
 #define LINK_IS_NOT_ADULT (gSaveContext.linkAge == AGE_FLAG_CHILD) 
  
 #define LINK_IS_ADULT (gSaveContext.linkAge == AGE_FLAG_ADULT) 
 #define LINK_IS_NOT_CHILD (gSaveContext.linkAge != AGE_FLAG_CHILD) 
```

```suggestion
#define AGE_REQ_ADULT AGE_FLAG_ADULT
#define AGE_REQ_CHILD AGE_FLAG_CHILD
```